src/main/java/com/jwt/mybatis/jwtApplication/config/AppConfig.java

```diff
@@ -0,0 +1,43 @@
+//package com.jwt.mybatis.jwtApplication.config;
```

Comment:
If a code is not being used please follow the practice of removing it instead of keeping the code commented.

Reply:
Already removed in latest commit

Reply:
Removed in the latest commit